README.md

```diff
@@ -1218,6 +1218,62 @@ Google service account. Defaults to `$GITLAB_OBJECT_STORE_CONNECTION_GOOGLE_CLIE
 
 Default Google key file. Defaults to `$GITLAB_OBJECT_STORE_CONNECTION_GOOGLE_JSON_KEY_LOCATION` (`/gcs/key.json`)
 
+##### `GITLAB_TERRAFORM_STATE_ENABLED`
+
+Enable/Disable Terraform State support. Defaults to `true`.
+
+##### `GITLAB_TERRAFORM_STATE_STORAGE_PATH`
+
+Directory to store the terraform state data. Defaults to `$GITLAB_SHARED_DIR/terraform_state`
+
+##### `GITLAB_TERRAFORM_STATE_OBJECT_STORE_ENABLED`
+
+Enables Object Store for Terraform state that will be remote stored. Defaults to `false`
+
+##### `GITLAB_TERRAFORM_STATE_OBJECT_STORE_REMOTE_DIRECTORY`
+
+Bucket name to store the Terraform state. Defaults to `terraform_state`
+
+##### `GITLAB_TERRAFORM_STATE_OBJECT_STORE_CONNECTION_PROVIDER`
+
+Connection Provider for the Object Store. (`AWS` or `Google`) Defaults to `$GITLAB_OBJECT_STORE_CONNECTION_PROVIDER` (`AWS`)
```

Comment:
Please change this sentence to: 

Connection Provider for the Object Store (`AWS` or `Google`).  Defaults to `$GITLAB_OBJECT_STORE_CONNECTION_PROVIDER` (i.e. `AWS`).